README.md

```diff
@@ -510,6 +521,7 @@ To create an issue in the github repository simple thing that shows a link.  Yee
 - [Requests](https://wwww.docs.python-requests.org): Used for online http requests/services
 - [wttr](http://wttr.in): Used for getting weather information
 - [file.io](https://file.io/): Used for fileshare
+- [HackerEarthAPI](https://www.hackerearth.com/docs/wiki/developers/legacy/): Used to run code
```

Comment:
You are using this version: https://www.hackerearth.com/docs/wiki/developers/v3/
But the link is legacy

Reply:
Sorry. It was a copy paste mistake. :/

Reply:
Shall I rectify and raise a PR for this?

Reply:
No it's okay. I'll fix it